### ISSUE TITLE
[Form] - unknown option row_attr

### ISSUE THREAD

**Issue by mcsky** [symfony/symfony#33682]
**Symfony version(s) affected**: 4.3.4

**Description**  
It seems that form field has the new option [row_attr](https://symfony.com/doc/current/reference/forms/types/form.html#row-attr). But I’m trying to use it (my version is 4.3.4 for all Sf dependencies) but I get the form error unknown option error.

```
The option "row_attr" does not exist. Defined options are: ........
```

**How to reproduce**  

Use the `row_attr` option

**Possible Solution**  

I fixed it by modifying the [FormType class](https://github.com/symfony/symfony/blob/4.3/src/Symfony/Component/Form/Extension/Core/Type/FormType.php) and add the option like done for `label_attr` for example.

Do I'm missing something? 
I can submit a PR, if this is a real issue

**Comment by stof** [comment]
it looks like the PR trying to add this option was incomplete.

**Comment by javiereguiluz** [comment]
I guess it's related to #33552, which is being fixed in #33573. Could you please check if the changes done in #33573 would solve the issue for you? Thanks!

**Comment by stof** [comment]
@javiereguiluz the issue is not only form themes. `form_div_layout.html.twig` has support for this `row_attr` view variable. but FormType does not support the option (and so does not set the variable either).
